pkg/kubelet/network/exec/exec.go

```diff
@@ -131,3 +159,39 @@ func (plugin *execNetworkPlugin) TearDownPod(namespace string, name string, id k
 	glog.V(5).Infof("TearDownPod 'exec' network plugin output: %s, %v", string(out), err)
 	return err
 }
+
+func (plugin *execNetworkPlugin) Status(namespace string, name string, id kubeletTypes.DockerID) (*network.PodNetworkStatus, error) {
+	out, err := utilexec.New().Command(plugin.getExecutable(), statusCmd, namespace, name, string(id)).CombinedOutput()
+	glog.V(5).Infof("Status 'exec' network plugin output: %s, %v", string(out), err)
+	if err != nil {
+		return nil, err
+	}
+	if string(out) == "" {
+		return nil, nil
+	}
+	findVersion := struct {
+		api.TypeMeta `json:",inline"`
+	}{}
+	err = json.Unmarshal(out, &findVersion)
+	if err != nil {
+		return nil, err
+	}
+
+	// check kind and version
+	if findVersion.Kind != "" && findVersion.Kind != "PodNetworkStatus" {
+		errStr := fmt.Sprintf("Invalid 'kind' returned in network status for pod '%s'. Valid value is 'PodNetworkStatus', got '%s'.", name, findVersion.Kind)
+		return nil, errors.New(errStr)
+	}
+	switch findVersion.APIVersion {
+	case "":
+		fallthrough
+	case "v1":
+		fallthrough
+	case "latest":
```

Comment:
what's the intent with "latest" ?  We don't do this for the main API, so let's keep this as simple as possible.